tests/ui/pages/desktop/base.py

```diff
@@ -11,7 +11,7 @@ class Base(Page):
 
     def __init__(self, selenium, base_url, locale='en-US', **kwargs):
         super(Base, self).__init__(
-            selenium, base_url, locale=locale, timeout=30, **kwargs)
+            selenium, base_url, locale=locale, timeout=10, **kwargs)
```

Comment:
Is that a performance optimization or why did we change that?

Reply:
This can be removed